### PR TITLE
Send data to Homepage and Post detail page

### PR DESCRIPTION
**Pull request by tienkane**
## Related Tickets:
- [#26824](https://edu-redmine.sun-asterisk.vn/issues/26824)
- [#26827](https://edu-redmine.sun-asterisk.vn/issues/26827)

## WHAT this PR does?
- Send data to Homepage
- Send data to Post detail page

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by tienkane** [comment]
ready

**Comment by dinhtienn** [comment]
ready

File: routes/web.php [hunk 1]
```diff
@@ -13,6 +13,16 @@
...
+    Route::get('/set-locale', 'FrontendController@setLocale')->name('set_locale');
+
+    Route::get('/bai-viet/{slug}', 'PostController@show')->name('post.detail');
+
+    Route::get('/danh-muc-chung', 'CategoryController@index')->name('category.overall');
+
+    Route::get('/danh-muc/{slug}', 'CategoryController@show')->name('category.detail');
+
+    Route::get('/tag/{name}', 'TagController@show')->name('tag.detail');
 });
```
Comment (huyenttm-0901) [inline routes/web.php]: dùng tiếng anh hết đi em

Comment (tienkane) [reply routes/web.php]: Done ạ

**Comment by tienkane** [comment]
ready

**Pull request merged**